.golangci.yml

```diff
@@ -64,7 +64,6 @@ linters-settings:
 linters:
   disable-all: true
   enable:
-    - depguard # Go linter that checks if package imports are in a list of acceptable packages [fast: true, auto-fix: false]
```

Comment:
this was never configured and it now complains about the imports we have, it's better to disable